front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -248,9 +253,8 @@ export function updateDaySinceDeparture(
           daySinceDeparture: currentDaySinceDeparture,
         };
       }
-      previousTime = departureInSeconds;
+      previousTime = isDepartureMidnight ? 0 : departureInSeconds;
```

Comment:
`isDepartureMidnight` is true if and only if `departureInSeconds === 0` so I think we can just keep the old logic here?

Reply:
you're right, changed